content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -1256,13 +1267,13 @@ any `set`or `atomic`array parent schemas make it impossible to unambiguously cor
 Here are some examples for transition rules:
 
 {{< table caption="Transition rules examples" >}}
-| Use Case                                                          | Rule
-| --------                                                          | --------
-| Immutability                                                      | `self.foo == oldSelf.foo`
-| Prevent modification/removal once assigned                        | `oldSelf != 'bar' \|\| self == 'bar'` or `!has(oldSelf.field) \|\| has(self.field)`
-| Append-only set                                                   | `self.all(element, element in oldSelf)`
-| If previous value was X, new value can only be A or B, not Y or Z | `oldSelf != 'X' \|\| self in ['A', 'B']`
-| Monotonic (non-decreasing) counters                               | `self >= oldSelf`
+| Use Case | Rule |
+| -------- | ---- |
+| Immutability | `self.foo == oldSelf.foo` |
+| Prevent modification/removal once assigned | `oldSelf != 'bar' \|\| self == 'bar'` or `!has(oldSelf.field) \|\| has(self.field)` |
+| Append-only set | `self.all(element, element in oldSelf)` |
+| If previous value was X, new value can only be A or B, not Y or Z | `oldSelf != 'X' \|\| self in ['A', 'B']` |
+| Monotonic (non-decreasing) counters | `self >= oldSelf` |
```

Comment:
I don't see the value of this change.

Reply:
Just add a closing `|`. The current table has a starting `|`, but no closing `|`.